app/frontend/Experiences/ExperienceBlockContainer/ExperienceBlockContainer.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+  AnnouncementBlock,
+  Block,
+  MultistepFormBlock,
+  ParticipantSummary,
+  QuestionBlock,
+} from '@cctv/types';
+
+import Announcement from '../Announcement/Announcement';
+import MultistepForm from '../MultistepForm/MultistepForm';
+import Poll from '../Poll/Poll';
+import Question from '../Question/Question';
+
+interface ExperienceBlockContainerProps {
+  block: Block;
+  participant: ParticipantSummary;
+}
+
+export default function ExperienceBlockContainer({
+  block,
+  participant,
+}: ExperienceBlockContainerProps) {
+  if (!participant) {
+    return <p>Participant information is missing.</p>;
+  }
+
+  if (!block.payload) {
+    return <p>Block configuration is missing.</p>;
+  }
```

Comment:
These are good for debugging, but I don't think folks will understand our underlying terminology. Maybe something like (kind of goofy, but I think funny error messages are good)

```suggestion
  if (!participant) {
    return <p>Do you even go here? You're not part of this!</p>;
  }

  if (!block.payload) {
    return <p>Party's over, everyone go home. Experience not found.</p>;
  }
```